addons/web/static/src/views/relational_model.js

```diff
@@ -2358,13 +2353,6 @@ export class DynamicGroupList extends DynamicList {
         if (!this.groups.length) {
             return;
         }
-        const everyGroupIsClosed = this.groups.every((group) => group.isFolded);
-        if (
-            everyGroupIsClosed &&
-            !(this.groupBy.includes(fieldName) || this.hasAggregate(fieldName))
```

Comment:
like fge said in the task's chatter (confirmed by dbo, see last message), this code should be kept. The problem is that this.groupBy is `[date:week]`  and fieldName is `date`. You should split the groupBy on ":" and take the first part, to remove the granularity. Note that as of 17, we only check the first level of groupBy (which makes sense), so I suggest something like
```js
        if (this.groups.every((group) => group.isFolded)) {
            if (this.groupBy[0].split(":")[0] !== fieldName) {
                if (!this.hasAggregate(fieldName)) {
                    return;
                }
            }
        }
```
which is very similar to what we do in the new model of 17.0